include/RAJA/config.hpp.in

```diff
@@ -218,12 +218,20 @@ const int COHERENCE_BLOCK_SIZE = @RAJA_COHERENCE_BLOCK_SIZE@;
 #if defined(RAJA_ENABLE_CUDA)
 #define RAJA_ALIGN_DATA(d)
 #else
-
 #define RAJA_ALIGN_DATA(d) __builtin_assume_aligned(d, DATA_ALIGN)
-
 #endif
 
-#define RAJA_SIMD  // TODO: Define this...
+#if defined(_OPENMP) && (_OPENMP >= 201307)
+#define RAJA_SIMD  _Pragma("omp simd")
+#define RAJA_NO_SIMD 
+#elif defined(__GNUC__) && defined(__GNUC_MINOR__) && \
+      ( ( (__GNUC__ == 4) && (__GNUC_MINOR__ == 9) ) || (__GNUC__ >= 5) )
+#define RAJA_SIMD    _Pragma("GCC ivdep")
+#define RAJA_NO_SIMD 
```

Comment:
@artv3 did you say gnu had an attribute to disable simd optimizations? Can we use that here?

Reply:
@rhornung67, GNU's attribute would have to be applied to the function not just the loop. It would look something like this: 

 template <typename Iterable, typename Func>
#if defined(RAJA_COMPILER_GNU)
RAJA_GNU_NO_SIMD
#else
RAJA_INLINE
#endif
void forall(const seq_exec &, Iterable &&iter, Func &&loop_body)
RAJA_INLINE void forall(const seq_exec &, Iterable &&iter, Func &&body) .

I recall I had to remove the inline statement for it to work properly. I can add it in and double check if this would be okay.

Reply:
Hmmm....Not sure I like that.  Leave as is for now. We should mention in the release notes for the tagged version that seq_exec doesn't necessarily disable simd optimizations when using the gnu compiler.